### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by randabt**
create program details fragment and layout
add necessary strings, styles and dimensions
add test button to programs layout in order to navigate from programs to program details

### CODE REVIEW

File: app/src/main/res/layout/fragment_program_details.xml [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <com.google.android.material.imageview.ShapeableImageView
+            android:id="@+id/image_profile"
+            style="@style/imageCircle"
+            android:src="@drawable/image"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+
```
Comment (davidmureithi) [inline app/src/main/res/layout/fragment_program_details.xml:27]: Please remove the space

File: app/src/main/res/layout/fragment_program_details.xml [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+            />
+
+        <TextView
+            android:id="@+id/programTitle"
+            style="@style/largeMuktaExtraBold"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/spacing_default"
+            android:text="Google Africa Scholarship Program"
+            app:layout_constraintBottom_toBottomOf="@+id/image_profile"
```
Comment (davidmureithi) [inline app/src/main/res/layout/fragment_program_details.xml:37]: Please format

File: app/src/main/res/layout/fragment_program_details.xml [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+
+        <TextView
+            android:id="@+id/aboutTv"
+            style="@style/mediumMukta"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/spacing_default"
+            android:text="@string/about"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/image_profile" />
```
Comment (davidmureithi) [inline app/src/main/res/layout/fragment_program_details.xml:50]: Format

File: app/src/main/res/layout/fragment_program_details.xml [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+                    android:layout_height="match_parent"
+                    >
+                    <TextView
+                        android:id="@+id/cardTitleMentorManager"
+                        style="@style/cardTitle"
+                        android:text="12"
+                        />
+
+                    <TextView
+                        android:id="@+id/cardDescriptionMentorManager"
```
Comment (davidmureithi) [inline app/src/main/res/layout/fragment_program_details.xml:90]: Please format this file. You can use CMD+OPT+L on a mac

File: app/src/main/res/navigation/app_navigation.xml [hunk 5]
```diff
@@ -18,7 +18,12 @@
     <fragment
         android:id="@+id/programsFragment"
         android:name="com.peculiaruc.alc_mmsystem_admin.ui.programs.ProgramsFragment"
```
Comment (davidmureithi) [inline app/src/main/res/navigation/app_navigation.xml:20]: Please format this file too. It helps make the review quick too.

File: app/src/main/res/layout/fragment_program_details.xml (hunk 1 above)
Comment (randabt) [reply app/src/main/res/layout/fragment_program_details.xml:27]: what space please? I format all files

**Review by davidmureithi — APPROVED** [review]

**Pull request merged**